### DISCUSSION TITLE
Calc app launches in background

### DISCUSSION THREAD

**Discussion by EddieUni** [SafeExamBrowser/seb-win-refactoring#619] [Q&A]
If I permit calc within SEB settings, it always runs in background and cannot be foregrounded (this seems to apply to all Windows store apps) - is there a workaround?

**Comment by dbuechel** [comment]
As already mentioned various times here in the forum, Windows Store (UWP) Apps are incompatible with SEB's kiosk modes. Background thereof is, that UWP Apps require the Windows shell (explorer.exe) to be running, which is not the case when SEB is running with either kiosk mode.
